progress/react/demo/src/index.css

```diff
@@ -0,0 +1,12 @@
+html,
+body,
+#root {
+    margin: 0;
+    padding: 0;
+    height: 100%;
+    width: 100%;
+    font-family: "Open Sans";
```

Comment:
We should be using a ThemeProvider and the PX Blue theme to get the correct font. It's a little overkill for a demo like this, but it helps to have all of our demos/projects set up in the same way for consistency. That was why I suggested using the PX Blue CLI to scaffold this since it does all of that stuff for free.

Reply:
I've added in the theme provider.  This should be doing everything the same as a CLI generated project now.